episodes/introduction.md

```diff
@@ -1,102 +1,95 @@
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- What can I expect from this Training?
+- What is expected of me in this Training?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with The Carpentries Workbench
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+By the end of this session, participants should be able to...
+
+- describe the format of the Trainer Training program and sessions.
+- summarise the lesson development experience of the participants in the cohort.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
-## Introduction
+:::::::::::::::::::::::::::::::::::::::::::::::::: checklist
 
-This is a lesson created via The Carpentries Workbench. It is written in
-[Pandoc-flavored Markdown](https://pandoc.org/MANUAL.txt) for static files and
-[R Markdown][r-markdown] for dynamic files that can render code into output. 
-Please refer to the [Introduction to The Carpentries 
-Workbench](https://carpentries.github.io/sandpaper-docs/) for full documentation.
+## Reading List
 
-What you need to know is that there are three sections required for a valid
-Carpentries lesson:
+- [The Carpentries Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html)
 
- 1. `questions` are displayed at the beginning of the episode to prime the
-    learner for the content.
- 2. `objectives` are the learning objectives for an episode displayed with
-    the questions.
- 3. `keypoints` are displayed at the end of the episode to reinforce the
-    objectives.
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
+## Introductions
 
-::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+The first discussion session will begin with a full round of Introductions,
+where each trainee will be asked to share some details about themselves
+and their past experience with curriculum development and The Carpentries.
 
-::::::::::::::::::::::::::::::::::::: challenge 
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-## Challenge 1: Can you do it?
 
-What is the output of this command?
+:::::::::::::::::::::::::::::::::::::::::::::::::: callout
 
-```r
-paste("This", "new", "lesson", "looks", "good")
-```
+## Curriculum is a work in progress
 
-:::::::::::::::::::::::: solution 
+Any lesson or curriculum has room for improvement,
+especially when it is relatively new and not extensively tested.
+The CLDT curriculum is still in the early stages of the development process:
+it has only been taught twice, both times by its primary authors.
```

Comment:
like this here but slight concern about forgetting to update it

Reply:
resolved in cf2666a